### ISSUE TITLE
CategoryServiceFactory.setConfigurationCategory options are not properly applied

### ISSUE THREAD

**Issue by jsamr** [mreuvers/typescript-logging#40]
I am trying to apply a config with custom logger to a specific category :

``` typescript
CategoryServiceFactory.setConfigurationCategory(integrityConf, intLog)
```
But it doesn't work. Should log to a file but doesn't. However, 
``` typescript
CategoryServiceFactory.setDefaultConfiguration(integrityConf)
```
does make it log to a file.

**Comment by mreuvers** [comment]
Hello @jsamr 

The latter does reset all existing categories and their existing loggers to use the new configuration by default.

The first call should do it for that specific category only, but not its child categories. 
I am not sure if that's the problem here, but if so try:

`CategoryServiceFactory.setConfigurationCategory(integrityConf, intLog, true)`

**Comment by jsamr** [comment]
So to sum it up, the specific configuration `integrityConf` does not apply to Category `intLog`. `CategoryServiceFactory.setConfigurationCategory` does not seems to honour its contract.

**Comment by jsamr** [comment]
@mreuvers Great! Looking forwards. Thanks for your reactivity.

**Comment by mreuvers** [comment]
Hello @jsamr 

I've published 0.6.3 in npm, it should have resolved this issue - could you please let me know if it does?

**Comment by jsamr** [comment]
@mreuvers Thanks a lot. I can confirm the patch fixed this issue for me!
Wish you all the best.
